README.md

```diff
@@ -58,6 +58,14 @@ Binaries are generated from `main` branch every night for `Linux` and `Windows`.
 
 Please be aware: nightly builds might have critical bugs, it's not recommended for use in production and no support provided.
 
+## K8s CI Dashboard
+
+The [k8s CI](https://testgrid.k8s.io/containerd) for containerd contains the following set of dashboards which is used to
+check the health of the kubernetes CRI aspect of containerd
+
+- [containerd-presubmits](https://testgrid.k8s.io/containerd-presubmits)
```

Comment:
let's leave the list out since it will possibly change over time at the group destination...